jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/EnvUtil.java

```diff
@@ -126,26 +127,22 @@ public static boolean greaterOrEqualsVersion(String versionA, String versionB) {
      */
     public static List<String[]> splitOnLastColon(List<String> listToSplit) {
         if (listToSplit != null) {
-            return Lists.transform(listToSplit, SPLIT_ON_LAST_COLON);
+            return listToSplit.stream().map(SPLIT_ON_LAST_COLON).collect(Collectors.toList());
         }
         return Collections.emptyList();
     }
 
-    private static final Function<String, Iterable<String>> COMMA_SPLITTER = new Function<String, Iterable<String>>() {
-        private Splitter COMMA_SPLIT = Splitter.on(",").trimResults().omitEmptyStrings();
-
+    private static final Function<String, List<String>> COMMA_SPLITTER = new Function<String, List<String>>() {
```

Comment:
Convert this single anonymous class to a lambda function

Reply:
Done